server/controllers/accountController.js

```diff
@@ -193,26 +199,25 @@ export default class authUsers{
                     status: 400,
                     message: 'only numbers are allowed in the account number field'
                 });
-            }
+            } else {
 
             if(req.body.status.toLowerCase() != 'active') {
```

Comment:
I think you could reduce lines 204 to 206 to a single line using logical operators.

Reply:
Also look into other parts of your code you could reduce the if statements by using logical operators.

Reply:
req.body.type.toLowerCase() != 'saving' && req.body.type.toLowerCase() != 'current'

changed at all the places that i found